rllib/core/learner/learner.py

```diff
@@ -522,14 +522,14 @@ def compile_results(
 
         # We put the stats for all modules under the ALL_MODULES key. e.g. average of
         # the gradients across all modules will go here.
-        mean_grads = [
-            np.mean(grad)
+        mean_abs_grads = [
```

Comment:
MEAN(ABS(grads)) is a much better metric than MEAN(grads) as the latter is always very close to zero, regardless of the nature of the gradients.

Reply:
what we should actually probably do here is the gradient norms. We can stick with this for now as it is similar, but in the future, norm is probably the way to go.